static-instrumenter/agent-instrumenter/src/main/java/io/opentelemetry/contrib/staticinstrumenter/advices/AgentAdvices.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter.advices;
+
+import io.opentelemetry.contrib.staticinstrumenter.internals.Main;
+import java.lang.instrument.Instrumentation;
+import net.bytebuddy.asm.Advice;
+
+@SuppressWarnings({"SystemOut", "CatchAndPrintStackTrace"})
+public final class AgentAdvices {
+  @SuppressWarnings("unused")
+  public static final class InstallBootstrapJarAdvice {
+
+    @Advice.OnMethodExit(suppress = Throwable.class, onThrowable = Throwable.class)
+    static void enterInstallBoostrapJar(
+        @Advice.Argument(value = 0, readOnly = false) Instrumentation inst) {
```

Comment:
This argument can be read-only, you're not overwriting it (same in the other advice class)